app/common/tables.html

```diff
@@ -0,0 +1,28 @@
+<div class="container margin">
+   <div class="minus"> </div>
+   <div class="minus"></div>
+   <div class="minus"></div>
+   <div class="textstyle">All Outlets</div>
+</div>
+<div class="container">
+   <div class="col-sm-12">
+      <div style="margin-top: 60px;">
+         <div class="color">{{tabledetails.tabletitle}}</div>
+         <table class="table table-condensed">
+            <thead>
+               <tr>
+                  <th class="headingcolor" ng-repeat="data in tabledetails.values">{{data}}</th>
+                  <th><a>{{tabledetails.alink.Text}}
+                     </a>
+                  </th>
+                  <th  ng-if="data.Type=='SUBMIT'" ng-repeat="data in tabledetails.button">
```

Comment:
yes but as per your directive, use tbody is empty??

if data is sent via directive how will u display it.